app/models/meetup.rb

```diff
@@ -33,6 +33,6 @@ def third_tuesday_of(time)
   validates_translated :title, :introduction, :conclusion, presence: true
 
   def self.next
-    order(starts_at: :asc).where('starts_at > ?', Time.now).first || NotScheduledYet.new
+    order(starts_at: :asc).where("starts_at > ?", Time.now).first || NotScheduledYet.new
```

Comment:
Use `find_by` instead of `where.first`.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.